### PR TITLE
Website Visual Updates & Fixing Broken Links

### PR DESCRIPTION
**Pull request by MaanavD**
### Description
Changes to carousel, links, headers, some font sizes, and images in the 'News' tab.

### Motivation and Context
Most changes were to better align the onnxruntime website to the onnx website (where relevant).

Preview link: https://maanavd.github.io/onnxruntime/index.html

### CODE REVIEW

File: footer.html [hunk 1]
```diff
@@ -39,7 +39,7 @@ <h3 class="footer-heading">Follow Us</h3>
                 </div>
                 <div class="col-6 col-sm-6 col-md-3">
                     <h3 class="footer-heading">Resources</h3>
-                    <ul class="list-unstyled text-small">
```
Comment (cassiebreviu) [inline footer.html:42]: It appears this change has added bullet points to the footer

Comment (cassiebreviu) [reply footer.html:42]: image

**Review by cassiebreviu — CHANGES_REQUESTED** [review]

File: footer.html (hunk 1 above)
Comment (MaanavD) [reply footer.html]: Fixed - that change has been reverted in favor of css changes

**Review by cassiebreviu — APPROVED** [review]

**Review by natke — DISMISSED** [review]
Please fix the broken links before merging: https://github.com/microsoft/onnxruntime/actions/runs/4019178252

**Comment by natke** [comment]
Something funky is going on with the install table

image

**Review by natke — APPROVED** [review]

**Pull request merged**